src/web_app/js/call.js

```diff
@@ -474,9 +503,14 @@ Call.prototype.startSignaling_ = function() {
     } else {
       this.pcClient_.startAsCallee(this.params_.messages);
     }
+    this.maybeReportGetUserMediaErrors_();
   }.bind(this))
   .catch(function(e) {
-    this.onError_('Create PeerConnection exception: ' + e.message);
+    this.onError_('Create PeerConnection exception: ' + e);
+    // Let the callstats backend know we failed connecting to the
+    // other endpoint.
+    this.pcClient_.sendCallstatsEvents('fabricFailed');
```

Comment:
I don't like having code duplication for this everywhere. The fail/terminate should happen in one place, e.g. hangup, or some cleanup function.